src/tests-bns/event-server-tests.ts

```diff
@@ -1067,12 +1067,5 @@ describe('BNS event server tests', () => {
     const configState = await db.getConfigState();
     expect(configState.bns_names_onchain_imported).toBe(false)
     expect(configState.bns_subdomains_imported).toBe(false)
-
-    await new Promise(resolve => setTimeout(async() => {
```

Comment:
I think this test is valuable, but what you're missing is that you're not invalidating the timeout once your test is complete.

You should keep a reference to the timeout so you can stop and invalidate it once the promise resolves.